### PR TITLE
Orbital: dismiss CardSecValInd restriction

### PR DESCRIPTION
**Pull request by jherreraa**
GWI-567

Remote:
122 tests, 509 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications 100% passed

Unit:
144 tests, 817 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications 100% passed

Rubocop:
760 files inspected, no offenses detected

### CODE REVIEW

**Review by sinourain — APPROVED** [review]
LGTM

File: lib/active_merchant/billing/gateways/orbital.rb [hunk 1]
```diff
@@ -594,7 +594,7 @@ def add_verification_value(xml, credit_card)
         #   Null-fill this attribute OR
         #   Do not submit the attribute at all.
         # - http://download.chasepaymentech.com/docs/orbital/orbital_gateway_xml_specification.pdf
-        xml.tag! :CardSecValInd, '1' if %w(visa discover diners_club).include?(credit_card.brand) && bin == '000001'
+        xml.tag! :CardSecValInd, '1' if %w(visa discover diners_club).include?(credit_card.brand)
```
Comment (Heavyblade) [inline lib/active_merchant/billing/gateways/orbital.rb:597]: 💬 Comment:

While checking the docs the only thing that seems odd, is that API docs version 9 says that CardSecValInd should be included when transacting with `visa`, `discover`, or `jbc` but here we have dinners_club.

Orbital_Gateway_XML_Interface_Developer_Guide



Should we update this section ?

**Review by Heavyblade — APPROVED** [review]
@jherreraa looks pretty much OK, but left a comment for you or maybe @naashton  consideration.

File: lib/active_merchant/billing/gateways/orbital.rb (hunk 1 above)
Comment (naashton) [reply lib/active_merchant/billing/gateways/orbital.rb:597]: This is a good call out. I think for this change, we'll leave it as is and raise this internally.

**Review by naashton — APPROVED** [review]
LGTM!

**Pull request merged**